lib/aws/AwsSigv4Signer.js

```diff
@@ -102,6 +103,9 @@ function AwsSigv4Signer(opts = {}) {
       }
 
       const currentCredentials = credentialsState.credentials;
+      // For AWS SDK V3 make sure token is valid at least until request has timed out.
+      const EXPIRATION_BUFFER_MS = toMs(options.requestTimeout || this.requestTimeout) * 2;
```

Comment:
It's not a constant, lowercase and maybe name it to something more specific, e.g. `credentialsExpireIn`?

Reply:
I will lowercase it, but personally I'd leave `buffer` in its name, so it could be, e.g. `tokenExpirationBuffer`.
Because `buffer` gives instant understanding of what's the purpose of this variable, unlike `credentialsExpireIn`.

Reply:
A buffer usually a space/memory thing. But not a hill I want to die on :)